conf/messages

```diff
@@ -477,8 +477,8 @@ update-contact-details.start-page.body.li4=proprietor
 update-contact-details.start-page.body.li5=partner
 update-contact-details.start-page.body.p2=If your role is on this list, you do not need additional authorisation.
 update-contact-details.start-page.body.h2=Before you start
-update-contact-details.start-page.body.p3=We'll ask you to confirm that you have authorisation before you submit any changes. We'll also need your name and telephone number.
-update-contact-details.start-page.body.p4=In some circumstances we'll ask for your agent codes for Self Assessment and Corporation Tax. The codes were sent by letter when your organisation asked for agent access to those services.
+update-contact-details.start-page.body.p3=We?ll ask you to confirm that you have authorisation before you submit any changes. We?ll also need your name and telephone number.
```

Comment:
what's this? Try shift + option + right square bracket

Reply:
I did use the smart quotes I'm not sure why it's changed to a '?'